### DISCUSSION TITLE
JCE Editor an Astroid News - stylesheets for editor content styling

### DISCUSSION THREAD

**Discussion by WM-Loose** [templaza/astroid-framework#376] [Show and tell]
Good News for all Astroid User:
**A custom.css file in Astroid framework templates is now automatically loaded along with standard template stylesheets for editor content styling.**

More infos:
https://www.joomlacontenteditor.net/news/jce-pro-2-9-40-released
BTW: Don't worry, it's just a typo in the JCE article and it means Astroid instead of Android ;-)

Have a nice day
Regards Dirk

**Comment by deltapapa01** [comment]
Hi,
this is really good news, thank you!

**Comment by franke12359** [comment]
It is possible to get a actual jce-editor.css file from Astroid system if the user press the "clear-cache" button?
Or is there anywhere a css file with the actual Astroid settings for the jce editor?
